install.sh

```diff
@@ -291,7 +291,10 @@ nvm_do_install() {
   local NVM_PROFILE
   NVM_PROFILE="$(nvm_detect_profile)"
   local INSTALL_DIR
-  INSTALL_DIR="$(nvm_install_dir)"
+  INSTALL_DIR="$(
+    nvm_install_dir | 
+    command sed s:^$HOME:\$HOME:
```

Comment:
That's a fair point. I'd prefer to have this change covered by automated tests, if possible.